ts/features/fci/screens/valid/FciDocumentsScreen.tsx

```diff
@@ -1,17 +1,162 @@
 import * as React from "react";
-import { SafeAreaView, ScrollView } from "react-native";
-import { H1 } from "../../../../components/core/typography/H1";
+import Pdf from "react-native-pdf";
+import { constNull, pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import { SafeAreaView, StyleSheet } from "react-native";
+import { useSelector } from "react-redux";
+import { useNavigation } from "@react-navigation/native";
+import IconFont from "../../../../components/ui/IconFont";
 import { IOStyles } from "../../../../components/core/variables/IOStyles";
 import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import { IOColors } from "../../../../components/core/variables/IOColors";
+import FooterWithButtons from "../../../../components/ui/FooterWithButtons";
+import I18n from "../../../../i18n";
+import DocumentsNavigationBar from "../../components/DocumentsNavigationBar";
+import TouchableDefaultOpacity from "../../../../components/TouchableDefaultOpacity";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import { fciSignatureDetailDocumentsSelector } from "../../store/reducers/fciSignatureRequest";
 
-const FciDocumentsScreen = () => (
-  <BaseScreenComponent goBack={true} headerTitle={"Documenti"}>
-    <SafeAreaView style={IOStyles.flex} testID={"FciDocumentsScreenTestID"}>
-      <ScrollView style={IOStyles.horizontalContentPadding}>
-        <H1>{"FciDocumentsScreen"}</H1>
-      </ScrollView>
-    </SafeAreaView>
-  </BaseScreenComponent>
-);
+const styles = StyleSheet.create({
+  pdf: {
+    flex: 1,
+    backgroundColor: IOColors.bluegrey
+  }
+});
 
+const FciDocumentsScreen = () => {
+  const pdfRef = React.useRef() as React.MutableRefObject<Pdf>;
```

Comment:
may this avoid the cast?

```suggestion
  const pdfRef = React.useRef<Pdf>(null);
```